### PR TITLE
LPS-190399 Method _commerceAddressLocalService.updateCommerceAddress(commerceAddress) was removed

### PR DESCRIPTION
**Pull request by michaelmdelcavalcanti**
Ticket: [LPS-190399](https://liferay.atlassian.net/browse/LPS-190399)

### CODE REVIEW

**Comment by michaelmdelcavalcanti** [comment]
ci:test:sf

**Comment by michaelmdelcavalcanti** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			Matcher matcher = _pattern.matcher(javaMethodContent);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java:112]: @michaelmdelcavalcanti, can we name this `pattern` in a more specific way as you've done in your previous pulls?

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+					javaMethodContent, content, matcher.group(2))) {
+
+				continue;
+			}
+
+			String line = matcher.group();
+
+			String newLine = StringUtil.replace(
+				line, matcher.group(1),
+				_newUpdateCommerceAddressImplementation(
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java:125]: @michaelmdelcavalcanti, don't forget: methods should start their names with verbs. Please rename to `getNewUpdateCommerceAddressImplementation`.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		String content, String fileContent, String variable) {
+
+		String variableTypeName = getVariableTypeName(
+			content, fileContent, variable);
+
+		return variableTypeName.equals("CommerceAddress");
+	}
+
+	private String _newUpdateCommerceAddressImplementation(
+		String variable, String indent) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java:67]: @michaelmdelcavalcanti, please sort the parameters alphabetically

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java (hunk 3 above)
Comment (michaelmdelcavalcanti) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java]: Done.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java (hunk 1 above)
Comment (michaelmdelcavalcanti) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java]: Done.

**Review by natocesarrego — APPROVED** [review]

**Pull request closed without merging**